spec/controllers/facility_statements_controller_spec.rb

```diff
@@ -133,19 +134,40 @@ def create_order_details
 
       it_should_deny_all [:staff, :senior_staff]
 
-      it "should create and send statements" do
-        expect(@order_detail1.reload.reviewed_at).to be < Time.zone.now
-        expect(@order_detail1.statement).to be_nil
-        expect(@order_detail1.price_policy).not_to be_nil
-        expect(@order_detail1.account.type).to eq(@account_type)
-        expect(@order_detail1.dispute_at).to be_nil
-
-        grant_and_sign_in(@user)
-        do_request
-        expect(flash[:error]).to be_nil
-        expect(assigns(:account_statements)).to have_key(@account)
-        expect(assigns(:account_statements)).to have_key(@account2)
-        expect(response).to redirect_to action: :new
+      context "when statement emailing is on", feature_setting: { skip_statement_emails: false } do
+        it "should create and send statements" do
```

Comment:
This and its opposite (lines 155-170) are nearly identical. Pulling the common parts out into shared examples would highlight the differences.